.gitignore

```diff
@@ -22,3 +22,9 @@ ehthumbs_vista.db
 # Visual Studio Code
 .vscode/
 .history/
+
+# Dependencies
+node_modules/
+
+# Compiled project files
+dist/
```

Comment:
Our convention is to only have those in gitignore.

The other should be in the global gitignore of the developer (as their are platform-dependent or IDE-dependent)

Plus, I'm wondering if `dist/` is not needed in case of actions?

Reply:
`gatling-gradle-plugin` was given with that... 
But ok for `gatling/gatling` one.

As this project is public and we will accept external contribution, let's let that here.


Thank for the explanation of the publish workflow